simple.js

```diff
@@ -496,6 +498,48 @@ define(function(require, exports, module) {
         }
 
         /**
+         * Detects Ace syntax mode from shebang for files with no extension
+         */
+        function detectSyntax() {
+            // syntax modes
+            var MODES = {
+                perl: /perl/,
+                php: /php/,
+                python: /python/,
+                ruby: /ruby/,
+                sh: /^(bash2?|ksh|sh)/
```

Comment:
Oh, not the same, not sure if C9 can syntax-highlight properly, just mentioning them as other shells. `zsh` too.